### ISSUE TITLE
Tensorboard not refreshing when checkpoints are saved

### ISSUE THREAD

**Issue by anooptoffy** [tensorflow/tpu#153]
I came across a minor issue while using TPU, I used buckets for storing checkpoints and used tensorboard for visualising the scalars involved in the training. But the loss graphs are not automatically updated. I have to manually kill the tensorboard and restart to see the new checkpoint information.

Can this scenario be fixed ?

**Comment by qiuminxu** [comment]
Hi Anoop, the scalar plugin should be able to automatically refresh for new check-in. Can you try git clone the latest TensorBoard from Github and see if the problem is resolved? If the issue persist, please file an issue [here](https://github.com/tensorflow/tensorboard/issues). Thanks.

**Comment by qiuminxu** [comment]
Hi anooptoffy@, were you able to have the issue fixed?

**Comment by qiuminxu** [comment]
Seems the same problem with https://github.com/tensorflow/tensorboard/issues/867
Can you try GCS_READ_CACHE_MAX_SIZE_MB=0 tensorboard --logdir foo suggested in the above issues?

**Comment by KennethKJ** [comment]
Is this issue still monitored despite being closed? Do I need to open a new issue?

I have encountered the same issue. I am using Tensorflow 1.13 in a Google Datalab notebook run on a vm on the compute engine. I initialize TensorBord the following way:

from google.datalab.ml import TensorBoard
TensorBoard().start("gs://mypath_to_estimator_output_dir")

I am assuming that google.datalab.ml is calling the TensorBoard version installed with TensorFlow 1.13, right?

An estimator is run on AI platform via gcloud and it stores its summary etc in a directory on Google Storage specified as its output dir and I start tensorboard specifying this output dir. It works but it never refreshes and I have to start a new instance to see the newest steps.

Thanks in advance for any help
